docs/fr/1-Experiences/Kubeflow.md

```diff
@@ -86,34 +91,74 @@ Cela vous permet de répondre à vos besoins de calcul tout en minimisant les
 
-<!-- prettier-ignore -->
-!!! note "Bogue de création de nœud lent."
-    En raison d'un bug avec le pare-feu, la création d'un nouveau nœud peut être très
-    lente dans certains cas (jusqu'à quelques heures). Un correctif pour ce problème est en cours.
+![Choisir CPU et RAM](../images/cpu-ram.PNG)
+
+
+## GPUs
+
+Si vous voulez un serveur GPU, sélectionnez « 1 » comme nombre de GPU et « NVIDIA » comme GPU
+fournisseur (le bouton de création sera grisé jusqu'à ce que le fournisseur de GPU soit sélectionné si
+vous avez un GPU spécifié). Les serveurs multi-GPU sont actuellement pris en charge sur l'AAW
+système uniquement sur une base spéciale sur demande, veuillez contacter les mainteneurs d'AAW si
+vous souhaitez un serveur multi-GPU.
+
+![Configuration GPU](../images/kubeflow_gpu_selection.PNG)
+
+Comme mentionné précédemment, si vous sélectionnez un serveur GPU, vous obtiendrez automatiquement 6 CPU
+cœurs et 112 Go de mémoire.
 
 <!-- prettier-ignore -->
 !!! note "Utilisez les machines GPU de manière responsable"
-    Il y a moins de machines GPU que de machines CPU, alors utilisez-les de manière responsable.
+     Les machines GPU sont nettement plus chères que les machines CPU,
+     alors utilisez-les de manière responsable.
 
-![Choisir CPU et RAM](../images/cpu-ram.PNG)
+## Volume d'espace de travail
 
-## Stockage de vos données
+Vous aurez besoin d'un volume d'espace de travail, sur lequel le dossier personnel sera monté. Là
+différentes options de configuration sont disponibles :
 
--Vous aurez envie de créer un volume de données ! Vous pourrez enregistrer votre
-travail ici, et si vous éteignez votre serveur, vous pourrez simplement remonter
-vos anciennes données en entrant le nom de votre ancien disque. **Il est
-important que vous vous souveniez du nom du volume.**
+- Vous pouvez soit réutiliser un volume d'espace de travail existant auparavant, soit en créer un nouveau.
 
-![Créer un volume de données](../images/kubeflow_volumes.png)
+- Vous pouvez spécifier la taille du volume de l'espace de travail, de 4 Gio à 32 Gio.
+
+![Créer un volume d'espace de travail](../images/workspace-volume.PNG)
 
 <!-- prettier-ignore -->
 !!! conseil "Vérifiez les anciens volumes en regardant l'option Existant"
-    Lorsque vous créez votre serveur vous avez la possibilité de réutiliser un ancien volume
-    ou en créer un nouveau. Vous souhaitez probablement réutiliser votre ancien volume.
+     Lorsque vous créez votre serveur vous avez la possibilité de réutiliser un ancien volume
+     ou en créer un nouveau. Vous souhaitez probablement réutiliser votre ancien volume.
+
+## Volumes de données
+
+Vous pouvez également créer des volumes de données pouvant être utilisés pour stocker des données supplémentaires. Plusieurs
+des volumes de données peuvent être créés. Cliquez sur le bouton Ajouter un nouveau volume pour créer un nouveau volume et
```

Comment:
extra "des" here i think. It sounds better as "Plusieurs volumes de données peuvent être créés".